pymeasure/instruments/santec/tsl500series.py

```diff
@@ -289,5 +292,47 @@ def sweep_routing(self, routing: SweepRouting):
     )
 
     sweep_count = Instrument.measurement(
-        ":WAVelength:SWEep:COUNt?", """Get the current number of completed sweeps."""
+        ":WAVelength:SWEep:COUNt?",
+        """Get the current number of completed sweeps.""",
+    )
+
+
+class TSL550(TSL570):
+    """Represents the Santec TSL-550 Tunable Laser and provides a high-level interface for
+    interacting with the instrument."""
+
+    def __init__(self, adapter, name="Santec TSL-550", **kwargs):
+        super().__init__(adapter, name, **kwargs)
+
+    @property
+    def command_set(self):
+        """The command set. The TSL-550 only accepts legacy commands.
```

Comment:
This docstring is probably the culprit for the failing test. As you get a value, it should start with "Get" (see the `command_set` property of the parent.